docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -107,7 +107,7 @@ $ newgrp libvirt
 [[xhyve-driver-install]]
 == xhyve Driver
 
-{project} is currently tested against *docker-machine-driver-xhyve* version 0.3.1.
+{project} is currently tested against *docker-machine-driver-xhyve* version 0.3.3.
```

Comment:
:-)

Reply:
There are other places also where you need to change it. 

=>  You can verify the existing version of the xhyve driver on your system using:

```
$ brew info --installed docker-machine-driver-xhyve
docker-machine-driver-xhyve: stable 0.3.1 (bottled), HEAD  => here also it need to be changed
Docker Machine driver for xhyve
https://github.com/zchee/docker-machine-driver-xhyve
/usr/local/Cellar/docker-machine-driver-xhyve/0.3.1 (3 files, 13.2M) * ==> Same here
  Poured from bottle on 2016-12-20 at 17:44:35
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/docker-machine-driver-xhyve.rb
```

=> Download the docker-machine-driver-xhyve binary using:

```
$ sudo curl -L  https://github.com/zchee/docker-machine-driver-xhyve/releases/download/v0.3.1/docker-machine-driver
```

Reply:
Made the changes in the two places.